third_party/build/bin/aioh2_build.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+#
+# Copyright 2021 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#build package from aioh2 master branch.
+
+set -e
+SCRIPT_DIR="$(dirname "$(realpath "$0")")"
+source "$SCRIPT_DIR"/../lib/util.sh
+PKGNAME=aioh2
+WORK_DIR=/tmp/build-"$PKGNAME"
+PWD1=$(pwd)
+
+function buildrequires() {
+    echo \
+        python3-venv \
+        python3-all \
+        debhelper \
+        dh-python \
+        python3-stem \
+        python3-stdeb \
+        python3-h2 \
+        python3-setuptools \
+        python3-distutils \
+        fakeroot
+
+}
+
+if_subcommand_exec
+
+sudo apt-get -y install \
```

Comment:
We can also remove this `apt-get install` I think

Reply:
Removed, since this script was restructured a little bit.